csm/PerceptionMan.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, network='ssd-mobilenet-v2', threshold=0.3):
         self.tracker = cv2.TrackerCSRT_create()
         # self.tracker = cv2.TrackerMOSSE_create()
         # self.tracker = cv2.TrackerMedianFlow_create()
+        pass
```

Comment:
What's this pass for?

Reply:
A relic from when I had to comment out your code to make Demo1 compile